proto/dwtools/abase/l0.test/Long.test.s

```diff
@@ -11781,51 +11781,79 @@ function longAllAreRepeated( test )
   /* Without evaluator */
 
   test.case = 'empty array';
-  var got = _.longAllAreRepeated( [] );
+  var src = [];
+  var got = _.longAllAreRepeated( src );
   var expected = true;
   test.identical( got, expected );
 
   test.case = 'identical numbers of array';
-  var got = _.longAllAreRepeated( [ 1, 1 ] );
+  var src = [ 1, 1 ];
+  var got = _.longAllAreRepeated( src );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'identical strings of array';
+  var src = [ 'string', 'string' ];
+  var got = _.longAllAreRepeated( src );
   var expected = true;
   test.identical( got, expected );
 
   test.case = 'single number of array';
-  var got = _.longAllAreRepeated( [ 1 ] );
+  var src = [ 1 ];
+  var got = _.longAllAreRepeated( src );
   var expected = false;
   test.identical( got, expected );
 
   test.case = 'some numbers of array are identical';
-  var got = _.longAllAreRepeated( [ 1, 2, 2 ] );
+  var src = [ 1, 2, 2 ];
+  var got = _.longAllAreRepeated( src );
   var expected = false;
   test.identical( got, expected );
 
-  /**/
-
-  test.case = 'identical strings of array';
-  var got = _.longAllAreRepeated( [ 'string', 'string' ] );
-  var expected = true;
+  test.case = 'some strings of array are identical';
+  var src = [ 'string', 'string', 'diff string' ];
+  var got = _.longAllAreRepeated( src );
+  var expected = false;
   test.identical( got, expected );
 
-  test.case = 'some strings of array are identical';
-  var got = _.longAllAreRepeated( [ 'string', 'string', 'diff string' ] );
+  test.case = 'complex data of array';
+  var src = [ 1, 2, [ 3 ], 'string', 6, 6 ];
+  var got = _.longAllAreRepeated( src );
   var expected = false;
   test.identical( got, expected );
```

Comment:
Test cases without callbacks and with callbacks should be the same. The difference between them should be in the callback and the result.
Please, correct test cases.

Routines `longAnyAreRepeated` and `longNoneAreRepeated` should be the same.